modules/rosa-sts-creating-cluster.adoc

```diff
@@ -90,15 +103,25 @@ You can configure the following default network IP ranges:
 
 For the CIDR-related `rosa` CLI arguments, see `rosa create cluster --help | grep cidr`. In the interactive mode, you are prompted for the settings.
 ====
-
-* To create a cluster with STS using interactive prompts:
++
+[NOTE]
+====
+The cluster state is `Pending` until the following steps are complete.
+====
++
+* To create a cluster with STS using the defaults:
 +
 [source,terminal]
 ----
-$ rosa create cluster --interactive
+$ rosa create cluster --cluster-name ${name} --sts
 ----
++
+[IMPORTANT]
+====
+To use the `--sts` option to create the cluster, you must use the default `ManagedOpenShift` prefix to create the account-wide roles and policies in the preceding step. The cluster will be created using the account roles and policies that include the default `ManagedOpenShift` prefix. 
```

Comment:
Actually, user doesn't need to use 'ManagedOpenShift' to create account-roles. When user create cluster with --sts it will go into interactive to let user choose the created roles. The created roles with command `rosa create account-roles` will be listed in the interactive mode.

Reply:
This comment looks like a duplicate of https://github.com/openshift/openshift-docs/pull/35762#discussion_r705028831.